speechbrain/nnet/CNN.py

```diff
@@ -1161,3 +1597,78 @@ def get_padding_elem_transposed(
         - 1
     )
     return int(padding)
+
+
+def gabor_impulse_response(t, center, fwhm):
```

Comment:
Wondering if these functions shouldn't go somewhere else, as they are not related to NN stuff, but more DSP ? what about Speechbrain.processing.features or speechbrain.process.signal_processing